src/codec_aom.c

```diff
@@ -871,8 +874,14 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
         if (!avifProcessAOMOptionsPostInit(codec, alpha)) {
             return AVIF_RESULT_INVALID_CODEC_SPECIFIC_OPTION;
         }
-        if (!codec->internal->tuningSet) {
-            if (aom_codec_control(&codec->internal->encoder, AOME_SET_TUNING, AOM_TUNE_SSIM) != AOM_CODEC_OK) {
+        if (!lossless && !codec->internal->tuningSet) {
+#if defined(AOM_HAVE_TUNE_IQ)
+            if (aomUsage == AOM_USAGE_ALL_INTRA &&
+                aom_codec_control(&codec->internal->encoder, AOME_SET_TUNING, AOM_TUNE_IQ) != AOM_CODEC_OK) {
```

Comment:
We are not ready to make `AOM_HAVE_TUNE_IQ` the default in libavif yet. But we can think about how to make the new tuning mode more discoverable.

For example, if we release libavif v1.2.0 after libaom v3.12.0 is released, we can mention the libaom codec-specific option tune=iq in the libavif v1.2.0 release notes.